src/algebra/group/units.lean

```diff
@@ -92,6 +98,12 @@ initialize_simps_projections add_units (val → coe as_prefix, neg → coe_neg a
   by change v = v' at e; subst v'; congr;
       simpa only [iv₂, vi₁, one_mul, mul_one] using mul_assoc i₂ v i₁
 
+@[to_additive, priority 0] instance : mul_one_class αˣ :=
```

Comment:
@eric-wieser , I incorporated your PR #11510 and that solves the timeout, too. But I need to lower the instance priority of `has_mul` and `mul_one_class` or some dot notation breaks, see the outdated comment above.

Do you understand what's happening here? And do you think that's any better than lowering the priority of `units.group`?